js/views/pageNavVw.js

```diff
@@ -78,12 +78,21 @@ module.exports = baseVw.extend({
 
     this.notificationsCl = new NotificationsCl();
     this.notificationsCl.comparator = function(notif) {
-      return -notif.get("timestamp");
+      return -notif.get('timestamp');
     };
-    this.notificationsFetch = this.notificationsCl.fetch();
+    this.notificationsFetch = this.notificationsCl.fetch({
+      data: {
+        limit: 6
```

Comment:
I decreased the amount of notifications in the first fetch, since this happens during initial page load when a lot of other stuff is happening. Subsequent requests will be for 10 per fetch.